R/turns.R

```diff
@@ -44,9 +44,7 @@ Turn <- new_class(
     ),
     text = new_property(
       class = class_character,
-      getter = function(self) {
-        paste0(unlist(lapply(self@contents, contents_text)), collapse = "")
-      }
+      getter = function(self) contents_text(self)
```

Comment:
You didn't change anything to make this work, right? You just noticed that we can now simplify the implementation?

Reply:
Right, I moved the implementation that was here into `method(contents_text, Turn)`.